src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -110,12 +110,7 @@ static void print_pci_info(std::ostream &ostr)
     if (pcidev::get_dev_total() != pcidev::get_dev_ready()) {
         ostr << "WARNING: "
             << "card(s) marked by '*' are not ready, is MPD runing? "
-            << "run 'systemctl status mpd' to check MPD details.";
-	    if (pcidev::get_dev_total(false) == 0)
-            ostr << std::endl;
-	    else
-            ostr << " please also run 'xbmgmt flash --scan --verbose' to further check card details."
-                << std::endl;
+            << "run 'systemctl status mpd' to check MPD details." << std::endl;
```

Comment:
let's just say that card is not ready, remove all guess work.

Reply:
Done